### ISSUE TITLE
Sustain node/edge addition mode

### ISSUE THREAD

**Issue by mrsfy** [iVis-at-Bilkent/chise.js#25]
_From @ugurdogrusoz on July 15, 2016 9:15_

We should think about how we can help the user to quickly successively create nodes of the same type. If no better solutions seems feasible, we can just keep the current addition mode upon creation rather than going back to the default selection mode.

_Copied from original issue: iVis-at-Bilkent/sbgnviz-js#154_

**Comment by ugurdogrusoz** [comment]
From Gary: omnigraffle has a nice implementation - click once for use shape, click button twice to shift it into a 'locked on' mode where the shape can be added multiple times.

**Comment by metincansiper** [comment]
I implemented this feature in the following way, if 'set-add-node-mode' button is clicked when 'add-node-mode' is already active then addition mode become sustainable. The same is implemented for 'add-edge-mode' too. 

When node/edge creation is sustainable the border color of related button groups become darker. The followings are screenshots from these button groups.

Default addition mode:
![default](https://cloud.githubusercontent.com/assets/6534865/16988519/5619a21e-4e99-11e6-9f04-9d9fa1aa32d1.PNG)

Sustainable mode:
![sustainable](https://cloud.githubusercontent.com/assets/6534865/16988532/61059156-4e99-11e6-855b-0ef3399a0aba.PNG)

**Comment by ugurdogrusoz** [comment]
Two improvements:
- Can we make the background of the icon in sustained mode a bit darker?
- When you click again on the icon when it's in the sustained mode, it should go toggle (back) to default (single) mode

**Comment by metincansiper** [comment]
- I made the background of the icon and dropdown in suntained mode a little bit darker. Also I reverted the changes I made on 'border-color' property of this button group for this issue. 
The new looks are as the following,

default:
![default](https://cloud.githubusercontent.com/assets/6534865/17056203/7ffd2240-501a-11e6-9e7a-7f1fb029b98a.PNG)

sustainable mode:
![sustainable](https://cloud.githubusercontent.com/assets/6534865/17056204/83338dbe-501a-11e6-8357-f092ff64155d.PNG)

- Now, it goes back to normal mode when we click again on the icon when it's in the sustained mode.
